spec.html

```diff
@@ -8790,6 +8832,26 @@ <h1>CreateBuiltinFunction ( _steps_, _internalSlotsList_ [ , _realm_ [ , _protot
       </emu-alg>
       <p>Each built-in function defined in this specification is created by calling the CreateBuiltinFunction abstract operation.</p>
     </emu-clause>
+
+    <emu-clause id="sec-createbuiltingeneratorfunction" aoid="CreateBuiltinGeneratorFunction">
+      <h1>CreateBuiltinGeneratorFunction ( _preludeSteps_, _steps_ [ , _iteratorKind_ [ , _generatorPrototype_ [ , _generatorInternalSlotsList_ ]]]) </h1>
+      <p>The abstract operation CreateBuiltinGeneratorFunction create a <dfn id="sec-builtin-generator-function">built-in generator function</dfn> object. The generator object it returns can have extra internal slots.</p>
+      <p>The abstract operation CreateBuiltinGeneratorFunction takes arguments _preludeSteps_, _steps_, and optional arguments _generatorInternalSlotsList_ (a List of internal slot names that the returned generator has), _generatorPrototype_ (the prototype of the returned generator, should inherit from %IteratorPrototype%). It performs the following steps when called:</p>
+      <emu-alg>
+        1. Assert: _preludeSteps_ is either a set of algorithm steps or other definition of a function's behaviour provided in this specification. It cannot contain Yield abstract operation.
```

Comment:
```suggestion
        1. Assert: _preludeSteps_ is either a set of algorithm steps or other definition of a function's behaviour provided in this specification. It cannot contain a Yield abstract operation.
```
what would "other definition" be for `preludeSteps`?

Reply:
The "other definition" is copied from `CreateBuiltinFunction`.

> 1. Assert: _steps_ is either a set of algorithm steps or other definition of a function's behaviour provided in this specification.